sdk/textanalytics/Azure.AI.TextAnalytics/src/Transforms.cs

```diff
@@ -159,11 +139,6 @@ internal static AnalyzeSentimentResultCollection ConvertToAnalyzeSentimentResult
 
         #region KeyPhrases
 
-        //internal static KeyPhraseCollection ConvertToKeyPhraseCollection(DocumentKeyPhrases documentKeyPhrases)
```

Comment:
sorry Jesse. :( 
The reason why we delete this is because we are using the same method that lives in `Transforms.Legacy` , right?

Reply:
No.... this was a mistake.    😄

Reply:
Restored.